tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -95,6 +95,31 @@ static PyObject * THPVariable_apply_(PyObject* self, PyObject* arg)
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject * THPVariable_sym_size(PyObject* self, PyObject* args, PyObject* kwargs) {
```

Comment:
@ezyang I took a shortcut here and added `sym_size` to test that I can get a `SymbolicIntNode` out of a tensor in python, I'm working on making we can just call `t.size()`. 
Unfortunately, there are some assumptions that `Size` objects can contain.

Reply:
sure np